src/gevent/pool.py

```diff
@@ -461,10 +461,13 @@ def __iter__(self):
         """
         return iter(self.greenlets)
 
-    def add(self, greenlet):
+    def add(self, greenlet, blocking=False, timeout=None):
         """
         Begin tracking the greenlet.
 
+        :keyword bool blocking: and :keyword bool timeout: are ignored
```

Comment:
This method probably needs to return True (but see below about return values; this may change), and the docstring needs a `.. versionadded::` section explaining the changes.

I have to think about whether the unused keywords are desirable here or not, but I'm leaning towards not. (Pool and Group are not fully equivalent to each other. According to the substitution principle a Pool is (should be) substitutable for a Group, but the reverse doesn't have to be true.)

Reply:
You're right, I missed returning True here. In any case, I agree with your broader point, so I'm going to remove the changes to `Group.add`.